netutils/interface.py

```diff
@@ -35,7 +36,7 @@ def _range_expand(regex_match):
                 number_range.append(int(value))
         return number_range
 
-    def _pairwise(interface_constant):
+    def _pairwise(interface_constant: t.List[int]) -> t.List[t.Iterable[t.Any]]:
         interface_constant_it = iter(interface_constant)
         return list(zip(interface_constant_it, interface_constant_it))
```

Comment:
If this really just wants to handle lists of integers, we can use the following signature:
```
def _pairwise(interface_constant: t.List[int]) -> t.List[t.Tuple[int, int]]:
```

If we need to handle arbitrary data types, which the `t.Any` in the return value suggests, we will need some [`TypeVar`](https://docs.python.org/3/library/typing.html#typing.TypeVar) usage here.